### DISCUSSION TITLE
Отпечатки на оружие как улика

### DISCUSSION THREAD

**Discussion by Zetik1** [octothorp-team/public-dobrograd#705] [Предложения]
Вероятно многие отыгрывая за полицейского сталкивались с тем, что находили на месте преступления или просто находили брошенный пистолет или подобное оружие, вы конечно расспросили у всех, никто обычно ничего не знает, ты берешь, упаковываешь его в зип-пакет, но что с ним делать дальше, в анализаторе он не проверяется, а через отыгровки найти человека - вероятность ну очень мала. Я предлагаю сделать так, что упакованное оружие в зип-пакет можно было проверять в анализаторе как улики с тела, чтобы можно было увидеть отпечатки тех людей, кто его держал в руках и отталкиваться уже от этих улик в расследовании дела.

- [x] да 
- [ ] нет, напишу в ответах к этому обсуждению почему

**Comment by Zetik1** [comment]
Я хер знает как здесь сделать голосование, извините если добавил не то

**Comment by LENFYS** [comment]
За

**Comment by travka4000** [comment]
Да да и да, я тоже за это гениально будет
